### PR TITLE
string concat for str.toLowerCase call

### PR DESCRIPTION
**Pull request by mrkshields**
Found and fixed a bug in #8

### CODE REVIEW

File: lib/popup.js [hunk 1]
```diff
@@ -254,6 +254,6 @@ function getExtension(lang) {
     case 'C++':
       return '.cpp'
     default:
-      return '.${lang.toLowerCase()}';
+      return '.' + lang.toLowerCase();
```
Comment (Jintin) [inline lib/popup.js:261]: Thanks for fixing this, given we have this we can remove Java, Go, Scala and Swift is it?

Comment (mrkshields) [reply lib/popup.js:261]: That's a good idea. I'll rework the code.

Comment (mrkshields) [reply lib/popup.js:261]: Reworked. I also added support for the rest of the supported Leetcode languages and sorted the case matches by name.

Comment (Jintin) [reply lib/popup.js:261]: Thanks

Comment (mrkshields) [reply lib/popup.js:261]: You're welcome

**Pull request merged**